src/Microsoft.Extensions.OData.Migration/MigrationOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Microsoft.Extensions.OData.Migration
+{
+    /// <summary>
+    /// Container for parameters to pass into middleware.
+    /// </summary>
+    public sealed class MigrationOptions

```

Comment:
Migration could become an overloaded term, I suggest you call it ODataMigrationOptions